heltour/tournament/tasks.py

```diff
@@ -20,18 +20,43 @@
 logger = get_task_logger(__name__)
 
 
+def just_username(qs):
+    return qs \
+        .distinct('lichess_username') \
+        .values('lichess_username')
+
+def active_player_usernames():
+    players_qs = Player.objects.all()
+    active_qs = players_qs.filter(seasonplayer__season__is_completed=False)
+    return [p['lichess_username'] for p in just_username(active_qs)]
+
+def not_updated_recently_usernames(active_usernames):
+    players_qs = Player.objects.all()
+    total_players = players_qs.count()
+    one_24th = total_players/24 # update them approximately every 24 hours
+    _24_hours = datetime.now() - timedelta(hours=24)
+    qs = players_qs \
+            .filter(date_modified__lte=_24_hours) \
+            .exclude(lichess_username__in=active_usernames) \
+            .order_by('date_modified')[:one_24th]
+    return [p['lichess_username'] for p in just_username(qs)]
```

Comment:
Yeah, that's a good point. It's still less than what we were previously including, but it's certainly not as good as it could be. I'll see if I can think of an easy way to exclude these.